### PR TITLE
dashing: init at 2018-02-15

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Motivation for this change

I want to use this. I'd love if this gets into 18.03 even, but if not I don't care (I can backport locally).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
@GrahamcOfBorg build dashing

**Comment by matthiasbeyer** [comment]
Added metadata. I _think_ this is MIT license, I'm not 100% sure though.

Also: I am not the maintainer for reasons I expressed in https://github.com/NixOS/nixpkgs/pull/38504

**Comment by matthiasbeyer** [comment]
For the record: Now I need to package `rsdocs-dashing` (a Rust tool) and then parsing rust documentations into dash docsets is possible with nixpkgs-installable tools :smile:

File: pkgs/tools/misc/dashing/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchgit {
+    inherit rev;
+    url = "https://github.com/technosophos/dashing";
+    sha256 = "066njyk3c1fqqr0v6aa6knp3dnksmh6hnl9d84fgd4wzyw3ma2an";
+  };
+
+  goDeps = ./deps.nix;
+
+  # TODO: add metadata https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes
+  meta = {
```
Comment (peterhoeg) [inline pkgs/tools/misc/dashing/default.nix]: Nitpick, change to ```meta = with stdenv.lib; {``` so we can get rid of the ```stdenv.lib``` bits below.

**Comment by peterhoeg** [comment]
Thanks @matthiasbeyer - now the next step is to create docsets for the nix ecosystem! ;-)

**Pull request merged**